core/src/main/java/org/kiwix/kiwixmobile/core/utils/files/FileUtils.kt

```diff
@@ -228,4 +236,15 @@ object FileUtils {
   } catch (e: IOException) {
     "".also { e.printStackTrace() }
   }
+
+  @JvmStatic fun isValidFile(filePath: String): Boolean {
```

Comment:
`isValidZimFile` naming would give more information about the function. What do you think?

Reply:
done